docs/Pitfalls.asciidoc

```diff
@@ -53,3 +53,45 @@ impacted by snapshot creation. Loading VM snapshots can also have an impact on
 SUT behavior as the execution of the first step after loading a snapshot might
 be delayed. This can lead to problems if the executed tests do not foresee an
 appropriate timeout margin.
+
+== DB migration from sqlite to postgreSQL
+In order to start using postgreSQL DBMS with openQA one need to perform following steps:
+- Install postgresql-server.
+- Adjust postgresql configuration file,
+namely `/var/lib/pgsql/data/pg_hba.conf`. For example:
```

Comment:
I'm not sure under which circumstances this should be necessary. I don't recall doing anything like this. Also you are not mentioning which entries are important in this file. I am afraid this will just confuse

Reply:
I had to add these, otherwise I was not able to connect

Reply:
in L66 add explicit permission for openqa database user to connect to all databases.

Reply:
@rwx788 adding the steps to set it up with the instructions I posted bewlow, removes the need of setting up or configuring anything in the pg_hba.conf